### PR TITLE
[WIP] Updates of arduino firmware for new gripper

### PR DESCRIPTION
**Pull request by ban-masa**
This PR should be tested with new grippers before merge.

### CODE REVIEW

File: jsk_2016_01_baxter_apc/firmware/gripper_front_manager_right.ino [hunk 1]
```diff
@@ -50,7 +50,7 @@ inline float rad_to_deg(float rad)
 
 void set_angle(void)
 {
-    myservo.write((int)(-rad_to_deg(servo_angle_msg.data) + 90.0));
+    myservo.write((int)(rad_to_deg(servo_angle_msg.data) - 90.0));
```
Comment (pazeshun) [inline jsk_2016_01_baxter_apc/firmware/gripper_front_manager_right.ino]: Is this for gripper-v3?
I think this shouldn't be fixed until right gripper becomes gripper-v3

**Comment by ban-masa** [comment]
Yeah, this PR is for gripper-v3.
I thought it wasn't a big problem because firmware of right arduino was already written.
Should I make PR for each arm separately?
In that case, I'll separate this PR.

**Comment by pazeshun** [comment]
I think separating is better.

**Comment by ban-masa** [comment]
OK.
Close this PR

**Pull request closed without merging**